prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```

Comment:
Hi Brandon, `flex-basis` goes on flex item elements, so try moving this to a rule set for the `.card-wrapper` class.

Reply:
Corrected
![prep-css-align-items-corrected](https://user-images.githubusercontent.com/109567499/183233755-9cf8792b-1128-4be6-81ea-1e94a0f432ff.PNG)
!